src/libmongoc/src/mongoc/mongoc-topology-private.h

```diff
@@ -187,6 +189,11 @@ typedef struct _mongoc_topology_t {
    bson_t *mongocryptd_spawn_args;
 #endif
 
+   // Corresponds to extraOptions.csflePath
+   mstr csfle_override_path;
+   // Corresponds to extraOptions.csfleRequired
+   bool csfle_required;
```

Comment:
The new options will need to be documented in `mongoc_auto_encryption_opts_set_extra.rst`.